src/mgr/ActivePyModules.h

```diff
@@ -189,6 +189,10 @@ class ActivePyModules
                   const std::string &notify_id);
   void notify_all(const LogEntry &log_entry);
 
+  auto& get_module_finisher(const std::string &name) {
```

Comment:
```suggestion
  auto& get_module_finisher(std::string_view name) {
```

Reply:
`modules` is a map. modules<std::string, std::shared_ptr<ActivePyModule>>. So it again needs to be converted to std::string which is not implicit. So let's stick to std::string reference ?

Reply:
sure